superset-embedded-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superset-ui/embedded-sdk",
-  "version": "0.1.0-alpha.9",
+  "version": "0.1.0-alpha.10",
```

Comment:
side note, when do you think we should we consider this not an alpha feature and bump the sdk to a stable version?

Reply:
I don't really have a strong opinion on this, but maybe it makes sense to do soon simply due to it being released for over a year now with no major issues?  cc @lilykuang who may have more context